src/transformers/tokenization_utils_base.py

```diff
@@ -111,6 +111,7 @@ class EncodingFast:
 SPECIAL_TOKENS_MAP_FILE = "special_tokens_map.json"
 ADDED_TOKENS_FILE = "added_tokens.json"
 TOKENIZER_CONFIG_FILE = "tokenizer_config.json"
+CONFIG_FILE = "config.json"
```

Comment:
No this file is the model configuration. It has nothing to do with the tokenizer and should not be put here.

Reply:
I agree. `AutoConfig.from_pretrained` makes this line unnecessary.